tools/importer/cmsis_imorter.py

```diff
@@ -0,0 +1,143 @@
+import os, json, stat, sys, shutil, errno, subprocess
+from os.path import dirname, abspath, basename, join
+
+ROOT = abspath(join(dirname(__file__), "../.."))
+CMSIS_REPO = "CMSIS_Repo"
+CMSIS_PATH = abspath(join(dirname(__file__), CMSIS_REPO))
+RTOS_UPDATE_BRANCH = "rtos_update"
```

Comment:
can we make it more useful, something like `feature_cmsis_rtx_{cmsis_sha_6char}`